cpp/src/IceGrid/RegistryI.cpp

```diff
@@ -510,9 +510,26 @@ RegistryI::startImpl()
     _wellKnownObjects->addEndpoint("Server", _serverAdapter->createDirectProxy(dummy));
     _wellKnownObjects->addEndpoint("Internal", _registryAdapter->createDirectProxy(dummy));
 
-    if (!setupUserAccountMapper())
+    //
```

Comment:
Moved `setupUserAccountMapper` implementation here as that was the only place we called.